### PR TITLE
revert texture renaming

### PR DESCRIPTION
**Pull request by z64a**
same as PR on pmret, do not modify texture names during dump or build

### CODE REVIEW

**Comment by bates64** [comment]
I'm generally in support of this if it is accepted in upstream: github.com/pmret/papermario/pull/1185. However, ideally, https://github.com/star-haven/papermario-dx/pull/57 gets out of development hell and we sync this change using the standard method, so I'm going to close this if thats OK. (Unless anyone else is so inclined, I intend to get to #57 once my university commitments are over.)

**Comment by z64a** [comment]
merging this so star rod can target a standard set of texture names

File: src/model.c [hunk 1]
```diff
@@ -2149,14 +2149,6 @@ void load_texture_by_name(ModelNodeProperty* propertyName, s32 romOffset, s32 si
             break;
         }
 
-        // try appending "tif" - this is a common issue with textures ported from Star Rod mods
-        char tifName[32];
-        strcpy(tifName, textureName);
-        strcat(tifName, "tif");
-        if (strcmp(tifName, header->name) == 0) {
-            break;
```
Comment (bates64) [inline src/model.c:2157]: Possibly worth keeping this and having it log an error for a migration period

Comment (z64a) [reply src/model.c:2157]: Texture filenames ending in tif is now the expected case, so this wouldn't do anything.

Comment (bates64) [reply src/model.c:2157]: image

**Review by bates64 — APPROVED** [review]

**Pull request merged**